README.md

```diff
@@ -1 +1,37 @@
-# Capstone1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Romanian Esports League 2022
+
+> This is a website created for the Romania Esports League 2022 competition.
+
+## Live demo
+
+https://vflorin1995.github.io/
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- Linters
+
+## Authors
+
+👤 **Voicu FLorin**
+
+- GitHub: [@vflorin1995](https://github.com/vflorin1995)
+- Twitter: [@vflorin1995](https://twitter.com/vflorin1995)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Design of the project is based on Cindy Shin design on Behance.
+- All other images are free or belong to their respective owners.
```

Comment:
- [ ] Great job acknowledging Cindy Shin 👍 . Please make sure you add a link to [her work](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),🙏